### ISSUE TITLE
Music-bot starts then stops

### ISSUE THREAD

**Issue by BlahBlah81** [DaMarine/Slabserver-Bugs#56]
### Server

#vctext

### Describe the bug

Music bot on occasion when joining a vc channel will quickly join and leave.

### Expected behaviour

I expected it to play the song i requested instead of it not playing and just joining and leaving.

### Steps to reproduce

'hard to reproduce sense its seemingly at random for me.

### Additional info

https://discord.com/channels/146701388234227712/147057212458532865/1025848442737659934 here is an example of it joining and leaving after i requested a song.

### Checklist

- [X] I have checked the Slabserver-Bugs repo to make sure this is not reported, or marked as "Won't Fix".

**Comment by chrissamael** [comment]
This happens whenever the musicbot has an error while fetching the video to play - I tried reproducing the issue with the same song, but it played just fine, so I think this was a temporary issue

I also checked if there's a new version of the music bot, but we are up2date on that front, too

TL;DR> most likely temporary hiccups, nothing we can really do (or even look at, no errors in the logs)

**Comment by DaMarine** [comment]
As it's a temporary hiccup I'm closing, if we continue to see a surge in these issues we'll look to reopen https://github.com/DaMarine/Slabserver-Staff/issues/133
